### DISCUSSION TITLE
what are tasks in our pull requests?

### DISCUSSION THREAD

**Discussion by sahildkun** [Spectrum-CETB/Spectober_Fest#27] [Q&A] [answered]
its showing 1 of 8 tasks  .....what is it and shall i be bothered

**Comment by Aditya-150** [comment, marked as answer]
**Tasks are generally the basic requirement of a pull request which need to be fulfilled in order to satisfy the project requirements. These requirements are generally set of instructions given by maintainers of the project in order to maintain quality standard in the project. All the task to be passed in not mandatory for a pull request to be merge ready. Tasks are represented by clickable checkboxes.** 

Some examples of tasks include: `Bug fixes`, `New feature update`, `Code of Conduct for the repo or organization`, `Project Guidelines` ant etc.

**Reply by Aditya-150** [reply]
@sahildkun is the doubt clear now. React to it to confirm from your side.

**Reply by sahildkun** [reply]
yes its clear. THANKS FOR THE CLARIFICATION

**Reply by Aditya-150** [reply]
@sahildkun Please `mark as answer` if it was helpful.🙃
